### DISCUSSION TITLE
Web login working

### DISCUSSION THREAD

**Discussion by schwobaseggl** [unikubehq/cli#49] [Announcements]
`unikube auth login` now triggers the browser to open so that users can login via the web frontend.

https://user-images.githubusercontent.com/14643239/122426153-4c766580-cf90-11eb-8111-c00f32da3b73.mp4

**Comment by Schille** [comment]
Very well done! :+1: 
Is it possible to close the browser window once the redirect was successful? - not sure if this is some browser-policy thing.
